### DISCUSSION TITLE
Python Demo

### DISCUSSION THREAD

**Discussion by vaughanknight** [Green-Software-Foundation/carbon-aware-sdk#38] [General]
DIscussion to go into details on what a python demo could look like (from  #27)  Things to discuss and brainstorm here:
* Demo scenario
* SKills needed
* Outcomes desired

**Comment by aradhika98** [comment]
Do we have any clarity on what we wish to achieve here? I am interested in contributing to this.

For Python, 
I was thinking of 2 main scenarios
1. Web Server
2. Machine Learning

We could plug in the sdk and check how different frameworks behave?
